app/code/Magento/PageCache/Model/App/Request/Http/IdentifierForSave.php

```diff
@@ -38,9 +48,11 @@ public function __construct(
      */
     public function getValue()
     {
+        $pattern = Identifier::PATTERN_MARKETING_PARAMETERS;
```

Comment:
This line is not extendable to append custom patterns. Dedicated service or public method visible to plugins would work better.

Reply:
@ilnytskyi, I moved it to separate method. Could you please review it?

Reply:
@engcom-Dash lgtm 

`getMarketingParameterPatterns` can be pluginized to add custom patterns, thanks